### ISSUE TITLE
Mongock should not create changeLog and lock collection when enabled = false

### ISSUE THREAD

**Issue by mjustin** [cloudyrock/mongock#225]
I was testing the 4.1.8 version of Mongock in Spring Boot, and I discovered that the Mongock collections were created (though empty) even when `spring.mongock.enabled` is `false`.  This is actually just fine for my use case, but it did surprise me.  Is it intended that the `mongockChangeLog` & `mongockLock` collections be created even when the functionality is disabled?

```
> show collections
[...]
mongockChangeLog
mongockLock
[...]
> db.mongockChangeLog.count()
0
> db.mongockLock.count()
0
```

### application.yml
```
  mongock:
    enabled: false
    change-logs-scan-package: com.example.my.migrations
    legacy-migration:
      collection-name: dbchangelog
```

### Log output:
```
17:10:01.567 [main] INFO  io.changock.runner.core.ChangockBase - Changock is disabled. Exiting.
```

**Comment by dieppa** [comment]
Actually no, that's a bug. mistakenly we didn't paid much attention to when Mongock is disabled. 

Thanks

**Comment by dieppa** [comment]
@mjustin fixed in 4.1.15

Please give it a try and let me know how it goes.

**Comment by leonelsanchesdasilva** [comment]
@dieppa I got to test this today (`application-test.properties` with `mongock.enabled=false`) and worked fine at 4.1.15.
